day_one/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use std::fs;
+fn main() {
+    let raw_input = fs::read_to_string("/Users/phil/Developer/aoc2024/day_one/src/input.txt")
```

Comment:
I would just use `include_str!('./input.txt')` here https://doc.rust-lang.org/std/macro.include_str.html

the compiler will error if the file does not exist, and it will always give you `&str` without needing to expect/unwrap. Supports relative paths easily too